packages/core/src/delegation/Delegation.ts

```diff
@@ -104,4 +104,50 @@ export default abstract class DelegationBaseNode
    * @returns Promise containing a submittable transaction.
    */
   public abstract revoke(identity: Identity): Promise<SubmittableExtrinsic>
+
+  /**
+   * Checks on chain whether a given identity is delegating to the current node.
+   *
+   * @param address The address of the identity.
+   * @returns An object containing a `node` owned by the identity if it is delegating, plus the number of `steps` traversed. `steps` is 0 if the address is owner of the current node.
+   */
+  public async isDelegating(
```

Comment:
Just a nitpick: the `is` in `isDelegating`, suggests a boolean return value, but it is a complex object.
I don't really have a much better name right now, though. Something with `find`? Like `findDelegatingNode`?

Reply:
mhhh yeah I meant to give it the same name as the function on the node, but of course the return type is odd for that. It was findParent before, so findDelegatingNode would work nicely

Reply:
or let's go super descriptive: `findAncestorOwnedBy(address)`

Reply:
uuuhhhh, I love it!